### ISSUE TITLE
Dealing with Django's CSRF Implications and API Usage.

### ISSUE THREAD

**Issue by ncrmro** [ncrmro/ango#6]
At the moment if you attempt to curl the graphql endpoint you will see a response indicating a missing CSRF issue.

```
curl -X POST -H "Content-Type: application/json" -d '{"query": "{ hello }"}' http://localhost:8000/graphql

<div id="summary">
  <h1>Forbidden <span>(403)</span></h1>
  <p>CSRF verification failed. Request aborted.</p>


  <p>You are seeing this message because this site requires a CSRF cookie when submitting forms. This cookie is required for security reasons, to ensure that your browser is not being hijacked by third parties.</p>
  <p>If you have configured your browser to disable cookies, please re-enable them, at least for this site, or for &#39;same-origin&#39; requests.</p>

</div>

```

This stackover flow post talks about the issue not being relevent to api usage, I'd like to use it if only for the sake of securing the admin section.

http://stackoverflow.com/questions/10741339/do-csrf-attacks-apply-to-apis

**Comment by ncrmro** [comment]
At the moment I've simply set the dev settings to remove the csrf middleware from the middlewares array.

**Comment by ncrmro** [comment]
Solved in dev enviorment incommit a0f1cfa73edf43fc398d289a5ef8fcbf04420092
